js/user_settings_dialog.js

```diff
@@ -60,7 +60,7 @@ function UserSettingsDialog(dialog, onLoad, onSave) {
         graphExpoOverride   : true,             // Ability to toggle Expo off=normal/ on=force 100%
         graphGridOverride   : true,             // Ability to toggle Grid off=normal/ on=force disabled
 		analyserSampleRate	: 2000/*Hz*/,  		// the loop time for the log
-		analyserHanning	    : false,  			// use a hanning window on the analyser sample data
+		analyserHanning	    : true,  			// use a hanning window on the analyser sample data
```

Comment:
General remark: This line uses tabs instead of the 4 spaces normally used in this project. Yes, this is not your change, but as in most projects, it is seen as polite if code is left 'at least as good as, if not better' than how it was found, so this could have been fixed as part of the pull request.